dotCMS/src/integration-test/java/com/dotcms/publishing/job/SiteSearchJobImplTest.java

```diff
@@ -531,9 +672,17 @@ public void test_MultilangContent_IndexingAllLanguages()
             jobDataMap.put(SiteSearchJobImpl.QUARTZ_JOB_NAME,
                     SiteSearchJobImpl.RUNNING_ONCE_JOB_NAME);
             jobDataMap.put(SiteSearchJobImpl.INCLUDE_EXCLUDE, "all");
+
+            List<String> langsToIndex = new ArrayList<>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'langsToIndex' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5303548)